oci/main.py

```diff
@@ -112,6 +125,13 @@ def system_tests(args):
 
     out.success("System tests completed successfully, congratulations!")
 
+def invoke_build_artifacts(args):
+    if args.daemon:
+        with open("/var/tmp/oci_build_artifcats_{}_.log".format(args.change), 'w') as f:
+            with daemon.DaemonContext(stdout=f,stderr=f):
+                build_artifacts(args)
+    else:
+        build_artifacts(args)
```

Comment:
This is poor way to integrate such change. It means that we need to duplicate this boilerplate for any new command.

If we have code daemonizing, it should run after parsing arguments in run(), before we invoke the command functions.

Reply:
can add boilerplate
def executer(cmd, args)
which will run cmd with args according to arg.daemon